docker/1.14/Dockerfile.cpu

```diff
@@ -0,0 +1,65 @@
+FROM ubuntu:18.04
+
+LABEL maintainer="Amazon AI"
+LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
+
+# See http://bugs.python.org/issue19846
+ENV LANG=C.UTF-8
+# Python won’t try to write .pyc or .pyo files on the import of source modules
+ENV PYTHONDONTWRITEBYTECODE=1
+ENV PYTHONUNBUFFERED=1
+ENV SAGEMAKER_TFS_VERSION="${TFS_SHORT_VERSION}"
+ENV PATH="$PATH:/sagemaker"
+ENV LD_LIBRARY_PATH='/usr/local/lib:$LD_LIBRARY_PATH'
+ENV MODEL_BASE_PATH=/models
+# The only required piece is the model name in order to differentiate endpoints
+ENV MODEL_NAME=model
+
+ARG PYTHON=python3
+ARG PIP=pip3
+ARG TFS_SHORT_VERSION=1.14
+
+# nginx + njs
+RUN apt-get update \
+ && apt-get -y install --no-install-recommends curl gnupg2 ca-certificates git wget vim build-essential zlib1g-dev \
+ && curl -s http://nginx.org/keys/nginx_signing.key | apt-key add - \
+ && echo 'deb http://nginx.org/packages/ubuntu/ bionic nginx' >> /etc/apt/sources.list \
+ && apt-get update \
+ && apt-get -y install --no-install-recommends nginx nginx-module-njs python3 python3-pip python3-setuptools \
+ && apt-get clean \
+ && rm -rf /var/lib/apt/lists/*
+
+# cython, falcon, gunicorn
+RUN \
+    ${PIP} install cython==0.29.12 falcon==2.0.0 gunicorn==19.9.0 gevent==1.4.0 requests==2.22.0
+
+COPY ./ /
+
+RUN ${PIP} --no-cache-dir install  --upgrade pip setuptools
+
+# Some TF tools expect a "python" binary
+RUN ln -s $(which ${PYTHON}) /usr/local/bin/python
+
+RUN curl https://tensorflow-aws.s3-us-west-2.amazonaws.com/MKL-Libraries/libiomp5.so -o /usr/local/lib/libiomp5.so
+RUN curl https://tensorflow-aws.s3-us-west-2.amazonaws.com/MKL-Libraries/libmklml_intel.so -o /usr/local/lib/libmklml_intel.so
+
+RUN curl https://tensorflow-aws.s3-us-west-2.amazonaws.com/1.14/Serving/CPU-WITH-MKL/tensorflow_model_server -o tensorflow_model_server && \
+chmod 555 tensorflow_model_server && cp tensorflow_model_server /usr/bin/tensorflow_model_server && \
+rm -f tensorflow_model_server
+
+RUN ${PIP} install -U --no-cache-dir awscli==1.16.196
```

Comment:
in 1.13 and 1.14 - cpu, we are doing
 `${PIP} install cython==0.29.12 falcon==2.0.0 gunicorn==19.9.0 gevent==1.4.0 requests==2.22.0`


`RUN ${PIP} --no-cache-dir install  --upgrade pip setuptools`


`RUN ${PIP} install -U --no-cache-dir awscli==1.16.196`

need to combine RUN statement 1&3 and the combine line should appear after 2

Reply:
Updated